Example/AccessibilitySnapshot/SwiftUIView+EmbedInHostingController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Copyright 2019 Square Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+import SwiftUI
+
+@available(iOS 13.0, *)
+extension View {
+
+    // should be removed again when AccessibilitySnapshot supports snapshotting
+    // native SwiftUI views, see AccessibilitySnapshot/issues/37
+    func toVC() -> UIViewController {
+        let viewController = UIViewController()
```

Comment:
Why do we need to wrap the hosting controller in another VC?

Reply:
mh 🤔  good point, will try it out if it's possible to snapshot the HostingController directly

Reply:
I removed the Container and it still results in the same Snapshot! 👍  Great finding!